fluent-syntax/src/serializer.js

```diff
@@ -249,6 +259,17 @@ function serializeSelectExpression(expr) {
   return parts.join("");
 }
 
+function serializeVariantList(expr) {
```

Comment:
This function should add the opening and the closing braces around the `VariantList`.